app/models/member.model.js

```diff
@@ -29,7 +29,7 @@ module.exports = (sequelize, DataTypes) => {
         },
         nickname: {
             type: DataTypes.STRING,
-            unique: true,
+            unique: false,
```

Comment:
Codacy has a fix for the issue: [Replace `············` with `↹↹↹↹`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9683172)
```suggestion
				unique: false,
```